internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -145,7 +162,19 @@ func (i *IngressControllerPostHook) Do(cluster CommonCluster) error {
 
 	namespace := global.Config.Cluster.Namespace
 
-	return i.helmService.ApplyDeployment(context.Background(), cluster.GetID(), namespace, config.Ingress.Chart, "ingress", valuesBytes, config.Ingress.Version)
+	err = a.helmService.ApplyDeployment(
+		context.Background(),
+		input.ClusterID,
+		namespace, config.Ingress.Chart,
+		"ingress",
+		valuesBytes,
+		config.Ingress.Version)
+
+	if err != nil {
+		return errors.WrapIf(err, "failed to merge treafik values with config")
```

Comment:
**Recommended**: I believe the error message here should rather be `failed to install/deploy [see my other comment] ingress controller` for improved clarity when debugging the operations.